x-pack/plugins/maps/public/connected_components/right_side_controls/layer_control/layer_control.tsx

```diff
@@ -190,3 +191,11 @@ export function LayerControl({
     </Fragment>
   );
 }
+
+export function LayerControl(props: Props) {
```

Comment:
Why is LayerControl wrapped in KibanaRenderContextProvider? Its only used in x-pack/plugins/maps/public/connected_components/right_side_controls/right_side_controls.tsx and this component is already in the react tree.

Reply:
These changes were pulled out of a larger draft PR that served to uncover test failures. It's been several builds since there was a test failure that was related to LayerControl. I need some time to go back and investigate which test was specifically failing around this, and then I'll update here.

Reply:
@nreese I backed out the LayerControl changes in the main PR and re-tested. I didn't see a build failure pop up. I must have been chasing a ghost 👻 when I made this change. Sorry about that, but thank you for the close attention to detail!

Backed out LayerControl changes from this PR: 3b52a4dd30e